tools/update-lib-configs.js

```diff
@@ -58,7 +58,7 @@ function formatCategory(category) {
 module.exports = {
   parser: require.resolve('vue-eslint-parser'),
   parserOptions: {
-    ecmaVersion: 2020,
+    ecmaVersion: "latest",
     sourceType: 'module'
   },
   env: {
```

Comment:
Should we also remove `env` to fix #1991 completely?

Reply:
Thank you for letting me know, I've changed it.